### PR TITLE
Add scopeinstance test

### PR DESCRIPTION
**Pull request by jmrodri**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,41 +4,41 @@ go 1.18
...
-	k8s.io/api v0.24.4
-	k8s.io/apimachinery v0.24.4
-	k8s.io/client-go v0.24.4
-	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	github.com/onsi/ginkgo/v2 v2.2.0
+	github.com/onsi/gomega v1.20.1
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
+	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
```
Comment (everettraven) [inline go.mod:12]: What caused these bumps? I'm happy to bump the k8s dependencies, but if we are we might as well bump `controller-runtime` to `v0.13.0`

**Review by everettraven — CHANGES_REQUESTED** [review]
A couple minor non-blocking nits (they can be done in a follow-up):
- Debug print statements removal
- Bump controller-runtime to `v0.13.0` since we are bumping k8s dependencies to `v0.25.0`

Only one blocking thing:
- Fix the linter failure

File: go.mod (hunk 1 above)
Comment (jmrodri) [reply go.mod:12]: Oh interesting. I'll double check what happened.

**Pull request merged**